static/app/types/index.tsx

```diff
@@ -2025,3 +2026,25 @@ export type KeyValueListData = {
   subjectDataTestId?: string;
   subjectIcon?: React.ReactNode;
 }[];
+
+export type ExternalActorMapping = {
+  id: string;
+  externalName: string;
+  memberId?: string;
+  teamId?: string;
+  sentryName: string;
```

Comment:
@NisanthanNanthakumar Wondering if `ExternalActorMapping` should be a union between two types: one with `memberId` and one with `teamId`. Is it possible `teamId` and `memberId` could be both missing or present?

Reply:
@scefali no, it's either or. I think the current type is ok. We can refactor in the future if this type becomes unwieldily.